src/main/java/org/minions/devfund/bruno/WhackAMole.java

```diff
@@ -0,0 +1,233 @@
+package org.minions.devfund.bruno;
+
+import java.util.Random;
+
+/**
+ * This Class in charge to manage the whack a mole game.
+ */
+public class WhackAMole {
+
+    private static final String BLANK_SPACE = " ";
+    private static final int ZERO = 0;
+    private Random random;
+    private String[][] moleGrid;
```

Comment:
In order to meet the requirements from edx this should be char
` moleGrid which is a 2-dimensional array of chars.`